test/test_docker/test_app.py

```diff
@@ -0,0 +1,88 @@
+import os
+import subprocess
+
+import gradio_client
+import pytest
+import requests
+
+file_dir = os.path.dirname(os.path.abspath(__file__))
+gradio_dir = os.path.dirname(os.path.dirname(file_dir))
+
+subprocess.run(
+    ["python", "-m", "build", "--wheel", gradio_dir, "--outdir", file_dir], check=True
+)
+subprocess.run(
+    [
+        "python",
+        "-m",
+        "build",
+        "--wheel",
+        os.path.join(gradio_dir, "client/python/"),
+        "--outdir",
+        file_dir,
+    ],
+    check=True,
+)
+
```

Comment:
Instead of reinstalling `gradio` and `gradio-client`, which might add build time, can you modify the github workflow files to add the `--outdir` command?